invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteCommunityModal.js

```diff
@@ -168,8 +180,10 @@ export class DeleteCommunityModal extends Component {
                     /* eslint-disable-next-line jsx-a11y/label-has-associated-control */
                     <label>
                       <Trans>
-                        <strong>All the members</strong> will be removed from the
-                        community.
+                        <strong>
+                          {membersCount ? membersCount.toString() : "All the"} members
```

Comment:
I am not sure how `<Trans>` deals with dynamic strings. E.g. this string could have any number or `All the` plus another string (that is always translated).

Since I am not an expert on this part, I am just leaving a note and will just let other jump in.

Also, I think the string `All the` can be removed since we're always passing the number of members to be deleted.

Reply:
ping @anikachurilova

Reply:
Looks valid for me. In any case, if not sure, you can always check how the string will be marked: https://inveniordm.docs.cern.ch/develop/howtos/i18n/#usage_1